docs/core/tools/global-tools-how-to-create.md

```diff
@@ -0,0 +1,197 @@
+           ......................'..'......    .......
+        .........................'..'.....       .......
+     ........    ..'.............'..'....      ..........
+   ..'..'...      ...............'.......      ..........
+  ...'......     ...... ..........  ......         .......
+ ...........   .......              ........        ......
+.......        '...'.'.              '.'.'.'         ....
+.......       .....'..               ..'.....
+   ..       ..........               ..'........
+          ............               ..............
+         .............               '..............
+        ...........'..              .'.'............
+       ...............              .'.'.............
+      .............'..               ..'..'...........
+      ...............                 .'..............
+       .........                        ..............
+        .....
+";
+    Console.WriteLine(bot);
+}
+```
+
+### Test the tool
+
+You can run the project and see the output. Try these variations to run our application:
+
+```csharp
+dotnet run
+dotnet run -- "Hello from the bot"
+dotnet run -- hello from the bot
+```
+
+All arguments after the `--` delimiter are passed to our application.
+
+## Setup the global tool
+
+Before you can pack and distribute the application as a Global Tool, you need to modify the project file. Open the `botsay.csproj` file and add three new XML nodes to the `<Project><PropertyGroup>` node:
+
+- \<PackAsTool>  
+[REQUIRED] Indicates that the application will be packaged for install as a Global Tool.
+
+- \<ToolCommandName>  
+[OPTIONAL] An alternative name for the tool, otherwise the command name for the tool will be named after the project file. You can have multiple tools in a package, choosing a unique and friendly name will help differentiate from other tools in the same package.
+
+- \<PackageOutputPath>  
+[OPTIONAL] Where the NuGet package will be produced. The NuGet package is what the .NET Core CLI Global Tools uses to install your tool.
+
+```xml
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <TargetFramework>netcoreapp2.1</TargetFramework>
+
+    <PackAsTool>true</PackAsTool>
+    <ToolCommandName>botsay</ToolCommandName>
+    <PackageOutputPath>./nupkg</PackageOutputPath>
+
+  </PropertyGroup>
+
+</Project>
+```
+
+Even though `<PackageOutputPath>` is optional, we'll use it in this example. Make sure you set it: `<PackageOutputPath>./nupkg</PackageOutputPath>`.
+
+Next, create a NuGet package for your application.
+
+```console
+dotnet pack
+```
+
+The `botsay.1.0.0.nupkg` file will be created in the folder identified by the `<PackageOutputPath>` XML value from the `botsay.csproj` file, which should be the `./nupkg` folder. This will make it easy for us to install and test. When you want to release a tool publicly, you would upload it to [https://www.nuget.org](https://www.nuget.org).
```

Comment:
"should" -> "in this case"

Reply:
no us

you would upload -> you'd upload or just upload